lib/foreman_tasks_core/shareable_action.rb

```diff
@@ -15,5 +15,11 @@ def plan(input)
         plan_action(SmartProxyDynflowCore::Callback::Action, callback, planned_action.output)
       end
     end
+
+    private
+
+    def on_proxy?
```

Comment:
We could probably use this on line 14 as well, right?

Reply:
Done